androidgen/src/main/java/com/azure/autorest/android/template/AndroidServiceAsyncClientTemplate.java

```diff
@@ -1,9 +1,11 @@
 package com.azure.autorest.android.template;
 
+import com.azure.autorest.model.javamodel.JavaContext;
 import com.azure.autorest.template.ServiceAsyncClientTemplate;
 
 public class AndroidServiceAsyncClientTemplate extends ServiceAsyncClientTemplate {
     private static AndroidServiceAsyncClientTemplate _instance = new AndroidServiceAsyncClientTemplate();
+
```

Comment:
why do we want to limit @Generated annotation to non-android client? If we don't want to support partial update in Android, disabling partial-update flag can do the same thing.

Reply:
android has a different core lib and different http client, they might not have these.